app/metrics.yaml

```diff
@@ -123,6 +123,19 @@ events:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-03-01"
+  total_uri_count:
+    type: counter
+    description: >
+      A counter of URIs visited by the user
```

Comment:
So this is not unique URIs visited, just all URIs?

Reply:
Also specify if this is per-session

Reply:
This is intended to be per-session, including reloads of the same URL. The intent was that it work similarly to Focus. https://github.com/mozilla-mobile/focus-android/blob/master/docs/Telemetry.md#uri-count

Reply:
I'll add those details to the yaml.